discovery/trends/v0.1/trends.md

```diff
@@ -0,0 +1,355 @@
+# "Fediscovery": Trends
+
+## Overview
+
+Specification identifier: `trends`
+
+The `trends` capability allows fediverse servers to query FASP for
+content, hashtags and links that are currently trending.
```

Comment:
Does an FASP always have to provide all three capabilities at the same time? Would it be possible to offer just trending hashtags?

Reply:
Interesting idea. I do not currently think we should split this up any further, but we definitely could in the future if there is demand.